src/Tribe/Editor/REST/V1/Endpoints/Single_Ticket.php

```diff
@@ -328,7 +328,7 @@ public function add_ticket( WP_REST_Request $request, $nonce_action ) {
 		 * @param object              $ticket      Ticket that was just added.
 		 * @param array<string,mixed> $ticket_data The body of the request.
 		 */
-		do_action( 'tribe_tickets_ticket_added', $post_id, $ticket_id, $ticket_data );
+		do_action( 'tribe_tickets_ticket_added', $post_id, $ticket, $ticket_data );
```

Comment:
I am not sure about this change here, since someone could be using this hook already, expecting an ID, and it suddenly is an object now?

Reply:
@dpanta94 the doc block needs to be updated here, the `$ticket_id` should be fine.

Reply:
@bordoni @rafsuntaskin `$ticket` is still the ticket id not an object. The difference is, that prior `$ticekt_id` would only get a value when the operation was about an update and not about create. `$ticket` stores the ticket id in both cases

Reply:
@dpanta94 https://github.com/the-events-calendar/event-tickets/pull/3200/files#r1734907208